portfolio/src/main/webapp/index.html

```diff
@@ -72,13 +74,15 @@ <h5> June 2017 - August 2017 </h5>
       <p> You can check out more of my work on 
           <a href = "https://github.com/shreyabarua" target = "_blank">GitHub</a>.</p>
 
-      <h2> Cooking </h2>
+      <h2 id = "cooking" style = "padding-top:70px;" > Cooking </h2>
```

Comment:
Nit (personal preference): Move the style attributes for `home`, `projects`, `cooking`, `gallery`, `contact` into style.css.

Reply:
okay, got it